src/pages/CDP.js

```diff
@@ -1,10 +1,13 @@
 import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
 import { hot } from 'react-hot-loader/root';
 import PageContentLayout from 'layouts/PageContentLayout';
 import lang from 'languages';
+import { getUsdPrice, calcCDPParams, cdpParamsAreValid } from 'utils/ui';
```

Comment:
Would be nice to re-use the calc methods, they were written initially with the cdp create flow in mind so the interface might not be ideal but feel free to adjust them to fit the page needs as well.